backend/server.py

```
@@ -7734,6 +7734,39 @@ async def rate_limit_stats(user: dict = Depends(get_current_user)) -> dict[str,
     return {"providers": get_tracker().get_stats()}
 
 
+@app.get("/api/metrics/self-heal")
+async def self_heal_stats(user: dict = Depends(get_current_user)) -> dict[str, object]:
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)